packages/client/hmi-client/src/components/workflow/ops/optimize-ciemss/tera-intervention-chart.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+	<vega-chart :visualization-spec="lineSpec" />
+</template>
```

Comment:
I don't see this component being used? 

And why are we flip-flopping between procedurally generated chart versus a light weight vue-component wrapper?  Was there a particular reason we are going back?

Reply:
Sorry this wasnt supposed to be here, must have gotten mixed up with some git issues I was having.  This is component is now removed